addons/armaos/functions/fnc_computer_addActionTurnOn.sqf

```diff
@@ -0,0 +1,77 @@
+/**
+ * Adds the "turn on" ACE interaction for a given computer object. Computer texture changes accordingly.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ *
+ * Results:
+ * None
+ */
+
+params ["_computer"];
+
+_batteryLevel = _computer getVariable "AE3_batteryLevel";
```

Comment:
private

Reply:
changed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763